app/helpers/ontologies_helper.rb

```diff
@@ -481,9 +481,9 @@ def dispaly_complex_text(definitions)
     html = ""
     definitions.each do |definition|
       if definition.is_a?(String)
-        html+= "<p>" + definition + "</p>"
+        html += '<p class="prefLabel">' + definition + '</p>'
       elsif definition.respond_to?(:uri) && definition.uri
-        html+= "<p>" + definition.uri + "</p>"
+        html+= '<a class="btn btn-sm btn-light ajax-modified-cls" href="' + definition.uri + '">' + definition.uri + '</a>'
```

Comment:
There is a view component to use instead of a plain string; this one: https://stageportal.lirmm.fr/lookbook/inspect/link_field/default

Reply:
Done 
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/ae3128e6-9220-442a-a7c8-25784e0e20a6)